CefSharp/IPopupFeatures.cs

```diff
@@ -15,56 +17,28 @@ public interface IPopupFeatures
         /// <value>
         /// The x coordinate.
         /// </value>
-        int X { get; }
-        /// <summary>
-        /// Gets the set.
-        /// </summary>
-        /// <value>
-        /// The x coordinate set.
-        /// </value>
-        int XSet { get; }
+        int? X { get; }
```

Comment:
Can you change these to Nullable<int>, should hopefully resolve the build error in the ref assembly project. 
Thanks

Reply:
Ah, Ok. I will get to that tomorrow.